### PR TITLE
Implement --path-to-elm-make

### PR DESCRIPTION
**Pull request by kachkaev**
Hi @jfairbank 👋, a new CLI option here 🙂 

Passing `--path-to-elm-make=/path/to/elm-make` would allow users choose a custom `elm-make` instance (e.g. the one installed locally). Otherwise, a globally available `elm-make` command will be used to compile Elm code, just as before.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -93,8 +98,9 @@ import sh from 'shelljs';
     } catch (err) {
       throw new Error(`Provided --project-dir \`${rawProjectDir}\` is not a directory.`);
     }
-    if (userModulePath.indexOf(`${projectDir}/`) !== 0) {
-      throw new Error(`File \`${userModulePath}\` must be located within --project-dir \`${rawProjectDir}\``);
+    if (userModulePath.indexOf(`${projectDir}/`) !== 0
+    && userModulePath.indexOf(`${projectDir}\\`) !== 0) {
```
Comment (jfairbank) [inline src/index.js]: Let's use the `path` module [`sep` helper](https://nodejs.org/api/path.html#path_path_sep).

Also, we could make this clearer with `startsWith`, which has node 6 support.

```js
if (userModulePath.startsWith(`${projectDir}${path.sep}`) {
```

**Review by jfairbank — CHANGES_REQUESTED** [review]
Looks good! One minor tweak needed please.

File: src/index.js (hunk 1 above)
Comment (kachkaev) [reply src/index.js]: Great point! I did not even know about `path.sep` – should have read Node.js basics before doing things 😅

**Comment by kachkaev** [comment]
Done. Also added Node 10 to Travis.

While making an edit, I noticed that most error messages end with a full stop while a couple of others don't. WDYT of also including this commit to this PR: https://github.com/kachkaev/run-elm/commit/d32b24a6d6bb7ef1b6168b129673a8e2ccf6ac26? Strictly speaking, this is a breaking change if someone's matching error messages in their code, but it could be negligible enough to land into 2.3. Our tests seem to be unaffected, which means that we should probably add more of them in future.

Feel free to squeeze in that commit yourself or simply merge this PR without it.

**Review by jfairbank — APPROVED** [review]

**Comment by jfairbank** [comment]
Thanks @kachkaev! Let's do a separate PR for fixing the error messages.

**Comment by kachkaev** [comment]
👌 
Will do now.

**Pull request merged**